src/components/Header/ListOfContents.js

```diff
@@ -1,21 +1,122 @@
-export default function ListOfContents(props) {
+import "./Header.css";
+import {
+  FaPhone,
+  FaHome,
+  FaListAlt,
+  FaUserCircle,
+  FaWindowRestore,
+} from "react-icons/fa";
+import { motion } from "framer-motion";
+
+export default function ListOfContents() {
+  const listVariants = {
+    visible: { transition: { staggerChildren: 0.1 } },
+  };
+
+  const itemVariants = {
+    hidden: { opacity: 0, scale: 0.5 },
+    visible: {
+      opacity: 1,
+      scale: 1,
+      transition: {
+        type: "spring",
+        damping: 15,
+        stiffness: 200,
+      },
+    },
+  };
   return (
-    <ul>
-      <li onClick={props.hideNav}>
-        <a href="#home">Home</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#about">About</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#skills">Skills</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#projects">Projects</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#contact">Contact</a>
-      </li>
-    </ul>
+    <div className="nav_menu" id="nav-menu">
+      <motion.ul
+        variants={listVariants}
+        initial="hidden"
+        animate="visible"
+        className="nav_list"
+        id="navlist"
+      >
+        <motion.li className="nav_item" variants={itemVariants}>
```

Comment:
I would create a list of content for the <li> and map over it in component.
Here is the example for the list.

```
const navItems = [
    {
      href: "#",
      icon: <FaHome />,
      text: "Home",
    },
    {
      href: "#about",
      icon: <FaUserCircle/>,
      text: "About",
    },
....
....
....
  ];
```